### PR TITLE
Crash details page: crash diagram + narrative

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17183

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Follow instructions in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
2. Because all of the cr3 columns the crash diagram component uses are currently `null` until we do a back fill in the new `crashes` table, to test you will need to manually copy over some fields to a record.
3. In your db client, find a crash record in the new `crashes` table. Now find that same crash in the old `atd_txdot_crashes` table. Copy over the following columns from `atd_txdot_crashes` to the record in `crashes`: `cr3_stored_flag` (now a boolean instead of a string) and `cr3_file_metadata`
4. Now if you go to the crash details page for that crash id you should see the crash diagram there. You should also see a button to download CR3 pdf though you wont actually be able to locally.
5. Observe that the crash narrative is also pulling from the new table

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/Crashes/CrashDiagram.js [hunk 1]
```diff
@@ -119,27 +124,27 @@ const CrashDiagram = props => {
...
-              href={"https://cris.dot.state.tx.us/"}
-              target={"_blank"}
-              rel="noopener noreferrer"
-            >
-              CRIS website
-            </a>{" "}
-            for the latest status of this crash.
-          </div>
         ) : (
+          // ) : props.isTempRecord ? (
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/CrashDiagram.js:128]: just commenting this out for now until we figure out how we are going to handle temp records in the new db

File: atd-vze/src/views/Crashes/CrashDiagram.js [hunk 2]
```diff
@@ -14,15 +14,20 @@ import axios from "axios";
 import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
 import { isDev } from "../../helpers/environment";
 
-const CrashDiagram = props => {
+const CrashDiagram = ({
+  crashId,
+  isCr3Stored,
+  isTempRecord,
+  cr3FileMetadata,
+}) => {
```
Comment (johnclary) [inline atd-vze/src/views/Crashes/CrashDiagram.js:22]: 🙏

**Review by johnclary — APPROVED** [review]
yep—this is perfecto 💯 

side note: i noticed that the `cr3_stored_flag` and `cr3_file_metadata` columns are missing from the `crashes_edits` table, which was not intentional. and like you mentioned, Rose, we can drop the `investigator_narrative_ocr` column. i will include those changes in my next batch of column migrations 👍 

🚢 🚢 🚢 🚢 🚢

File: atd-vze/src/views/Crashes/Crash.js [hunk 3]
```diff
@@ -137,6 +134,9 @@ function Crash(props) {
     address_primary: primaryAddress,
     address_secondary: secondaryAddress,
     crash_injury_metrics_view: { years_of_life_lost: yearsOfLifeLost },
+    investigator_narrative: investigatorNarrative,
+    cr3_stored_flag: cr3StoredFlag,
+    cr3_file_metadata: cr3FileMetadata,
   } = crashData?.crashes_by_pk ? crashData?.crashes_by_pk : {};
```
Comment (chiaberry) [inline atd-vze/src/views/Crashes/Crash.js:140]: this is what i was mentioning in dev sync, that the nested entries will error out when its the fallback `{}` .

**Review by chiaberry — APPROVED** [review]
great test steps, I see the diagram and a button!

File: atd-vze/src/views/Crashes/Crash.js (hunk 3 above)
Comment (johnclary) [reply atd-vze/src/views/Crashes/Crash.js:140]: thanks for flagging this Chia! i have a fix staged in my ID column branch.

**Comment by johnclary** [comment]
I am merging this!

**Pull request merged**